tests/js-modules/modules.py

```diff
@@ -94,6 +94,29 @@ def generate_and_verify_jwk(client):
         assert body["kty"] == "RSA"
         assert body == ref_pub_jwk, f"{body} != {ref_pub_jwk}"
 
+    # EdDSA
+    priv_pem, pub_pem = infra.crypto.generate_eddsa_keypair()
+
+    # Private
+    ref_priv_jwk = jwk.JWK.from_pem(priv_pem.encode()).export(as_dict=True)
+    r = client.post(
+        "/app/eddsaPemToJwk", body={"pem": priv_pem, "kid": ref_priv_jwk["kid"]}
+    )
+    body = r.body.json()
+    assert r.status_code == http.HTTPStatus.OK
+    assert body["kty"] == "OKP"
+    assert body == ref_priv_jwk, f"{body} != {ref_priv_jwk}"
+
+    # Public
+    ref_pub_jwk = jwk.JWK.from_pem(pub_pem.encode()).export(as_dict=True)
+    r = client.post(
+        "/app/pubEddsaPemToJwk", body={"pem": pub_pem, "kid": ref_pub_jwk["kid"]}
+    )
+    body = r.body.json()
+    assert r.status_code == http.HTTPStatus.OK
+    assert body["kty"] == "OKP"
+    assert body == ref_pub_jwk, f"{body} != {ref_pub_jwk}"
```

Comment:
As a quick sanity check, could you run this test in a loop for at least 1000 times locally? This is because we had a surprising bug with the initial JWK conversion (see https://github.com/microsoft/CCF/pull/4388). Looking at the OpenSSL functions you're calling to do the conversion, I believe it'll be absolutely fine but just checking to avoid a spurious CI failure down the line would be great.

Reply:
It's an interesting bug, but fortunately the test passed locally 1000 times!